core/src/main/java/com/testaem/aem/core/models/impl/MultipleNewsImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.testaem.aem.core.models.impl;
+
+import com.testaem.aem.core.models.MultipleNews;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.Self;
+import org.apache.sling.models.annotations.injectorspecific.SlingObject;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@Model(
+        adaptables = SlingHttpServletRequest.class,
+        defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL,
+        adapters = MultipleNews.class
+)
+public class MultipleNewsImpl implements MultipleNews {
+
+    private List<String> resources = new ArrayList<>();
```

Comment:
(naming) Please see this is not a collection of Resource objects. This is a list of strings. A string is not a resource. Better call this `resoursePaths` to reduce possible confusion